autoscheduler/frontend/src/FrontendWorkspace.code-workspace

```diff
@@ -0,0 +1,20 @@
+{
+	"folders": [
+		{
+			"path": "."
+		},
+		{
+			"path": "../../../autoscheduler"
+		}
+	],
+	"settings": {
+		"eslint.run": "onSave",
+		"eslint.autoFixOnSave": true,
+		"eslint.validate": [
+			"javascript",
+			"javascriptreact",
+			{"language": "typescript", "autoFix": true},
+			{"language": "typescriptreact", "autoFix": true}
+		]
+	}
+}
```

Comment:
We're not going to include text-editor related configs in the project, as we're trying to let people use whatever they want (I know Scott uses Atom). Although, Ryan and I do both of course recommend VSCode. 

Instead, add add the file to the Wiki [here](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/wiki/VS-Code-Setup), with a bit of steps and background on what adding it will do.

Reply:
Likewise, remove the file from git and add it `.code-workspace` to the `.gitignore` when you get the chance 😄

Reply:
I've updated the Wiki and explained how to set up ESLint in the `.vscode` settings so that no changes would have to be made to the `.gitignore`

Reply:
Looks good!